.github/workflows/ci.yml

```diff
@@ -0,0 +1,145 @@
+# This file is part of REANA.
+# Copyright (C) 2020 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+name: CI
+
+on:
+  push:
+    branches: master
+  pull_request:
+    branches: master
+
+jobs:
+  Lint-shellcheck:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Runs shell script static analysis
+        run: |
+          sudo apt-get install shellcheck
+          ./run-tests.sh --check-shellscript
+
+  Lint-black:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Check Python code formatting
+        run: |
+          pip install black
```

Comment:
We are getting an unsupported black version for REANA (we pin to less than `20` see [here](https://github.com/reanahub/pytest-reana/blob/3a2f8b49462538ce6424b033548e6853dfa7e60e/setup.py#L37)) (cc @ParthS007 as [R-Client PR](https://github.com/reanahub/reana-client/pull/437/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR32) is also affected):
```
$ pip install black | grep "black-20"
Processing /Users/rodrigdi/Library/Caches/pip/wheels/95/a4/59/10cd5378d52f92cdb45025f040e4686e10ae5217961c25fd66/black-20.8b1-py3-none-any.whl
Successfully installed black-20.8b1
```
We will need to:
- Pin `black`
or
- Install `pytest-reana` instead

I think installing `pytest-reana` is the best option in the long term so the `black` dependency is controlled there only. For `maint-X.Y` branches we would need to change this file to install `pytest-reana<X+1` cc @tiborsimko

Reply:
I will make the change in `r-client` PR as well 👍